src/main/java/me/totalfreedom/totalfreedommod/command/Command_wipeworldguardregions.java

```diff
@@ -0,0 +1,77 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import com.sk89q.worldguard.bukkit.RegionContainer;
+import com.sk89q.worldguard.bukkit.WorldGuardPlugin;
+import com.sk89q.worldguard.protection.managers.RegionManager;
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+import java.util.Map;
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+
+@CommandPermissions(level = Rank.TELNET_ADMIN, source = SourceType.BOTH, blockHostConsole = false)
+@CommandParameters(description = "Wipe all Worldguard regions.", usage = "/<command> <world>", aliases = "wiperegions")
+public class Command_wipeworldguardregions extends FreedomCommand

```

Comment:
What is up with this class name? Way too long, the alias should be removed and the class should be renamed to Command_wiperegions